docs/preview/gha-runner-scale-set-controller/README.md

```diff
@@ -219,6 +219,25 @@ To fix this, you can either:
 
 ## Changelog
 
+### v0.4.0
+
+#### Major changes
+
+1. Surface EphemeralRunnerSet stats to AutoscalingRunnerSet [#2382](https://github.com/actions/actions-runner-controller/pull/2382)
+1. Remove list/watch secrets permission from manager cluster role
```

Comment:
I think we should have instructions for this upgrade i.e. how folks should go about it without breaking their environments